### ISSUE TITLE
Disable navigation in shared notes

### ISSUE THREAD

**Issue by anantone** [zadam/trilium#3223]
### Describe feature

Simply to disable navigation (including the parent and child notes links) in shared notes. Is that possible by editing the CSS? 

This is for a literary project where readers should only see the text of the note, which includes links to other notes, but not have other navigation options beside those links. 

Thank you!

**Comment by zadam** [comment]
Yes, you can define a `~shareCss` relation to a CSS note where you can hide all the elements you want.

https://github.com/zadam/trilium/wiki/Sharing

**Comment by anantone** [comment]
Well, thank you, I just didn't know it was so easy to hide the elements... 
Turns out all it takes is:
`#[id] {
	display: none;
}`

The `~shareCss` relation is working wonders now, thank you!
